### ISSUE TITLE
Make multi-arch images publicly available

### ISSUE THREAD

**Issue by spr-mweber3** [zalando/skipper#2114]
Hey there, I'd like to know why the multi-arch images that you build are not made publicly available? We'd like to utilize them in our mixed amd64/arm64 setup without having to worry what image with which tag gets deployed where.

Is there any specific reason why you're not storing the multi-arch images in the same registry as your other images?

**Comment by szuecs** [comment]
@spr-mweber3 the problem is that our opensource registry does not support multi-arch containers

**Comment by spr-mweber3** [comment]
GitHub Container Registry (ghcr.io) [supports multi-arch images](https://github.blog/changelog/2020-09-17-github-container-registry-support-for-multi-arch-images/) for almost two years now. Maybe this could be an option for you? I think it makes perfectly sense since you're hosting the code here already, too.

It's pretty straight-forward to set-up.

**Comment by szuecs** [comment]
I don't see there the docs for setting it up. Maybe you want to create a PR for this?

**Comment by MustafaSaber** [comment]
Hi @spr-mweber3 , I'm giving it a try let's see how it will go. We don't mind creating Github workflow for it. so feel free to edit on the mentioned PR.

**Comment by spr-mweber3** [comment]
Thanks for trying to progress on the matter. I'm not sure if having a separate flow in addition to your central delivery makes a whole lot of sense. I think it's easier integrated there. Nevertheless, I'll see to provide a PR with a GitHub Workflow that we use ourselves.

**Comment by MustafaSaber** [comment]
Hi @spr-mweber3 can you test the current public images in your setup?

```
docker pull ghcr.io/zalando/skipper:v0.14.2
docker pull ghcr.io/zalando/skipper-arm64:v0.14.2
docker pull ghcr.io/zalando/skipper-armv7:v0.14.2
```

**Comment by spr-mweber3** [comment]
With `docker manifest inspect --verbose ghcr.io/zalando/skipper:v0.14.2` I can see that I can pull the image but it's not multi-arch. It's `amd64` only. But I guess you're doing one step after another here.

**Comment by spr-mweber3** [comment]
Awesome work, thank you @MustafaSaber ... really appreciate it.

**Comment by MustafaSaber** [comment]
urw! It's merged now with the right tag  🥳

**Comment by MustafaSaber** [comment]
Closed with #2173
